h/admin.py

```diff
@@ -185,6 +185,38 @@ def users_index(request):
     return {'username': username, 'user': user}
 
 
+@view.view_config(route_name='admin_badge',
+                  request_method='GET',
+                  renderer='h:templates/admin/badge.html.jinja2',
+                  permission='admin')
+def badge_index(_):
+    return {"uris": models.Blocklist.all()}
+
+
+@view.view_config(route_name='admin_badge',
+                  request_method='POST',
+                  request_param='add',
+                  renderer='h:templates/admin/badge.html.jinja2',
+                  permission='admin')
+def badge_add(request):
+    try:
+        request.db.add(models.Blocklist(uri=request.params['add']))
+    except ValueError as err:
+        request.session.flash(err.message, 'error')
+    return badge_index(request)
+
+
+@view.view_config(route_name='admin_badge',
+                  request_method='POST',
+                  request_param='remove',
```

Comment:
This is inconsistent with all the other remove views in this file. I'm sorry if this is nitpicking but inconsistency is just avoidable overhead when trying to understand the code.